wasp/apps/snake.py

```diff
@@ -0,0 +1,219 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright (C) 2020 Johannes Wache
+
+# 2-bit RLE, generated from res/snake_game.png, 856 bytes
```

Comment:
As with previous PR, we need res/snake_game.png to be included as well as the resulting RLE code.